pywinauto/linux/atspi_objects.py

```diff
@@ -507,6 +530,21 @@ class _AtspiAccessible(Structure):
 ]
 
 
+def g_error_handler(func):
+    def wrapper(*args, **kwargs):
+        error = _GError()
+        ep = POINTER(POINTER(_GError))(error)
+        kwargs["g_error_pointer"] = ep
+        res = func(*args, **kwargs)
+        if ep and ep.contents:
+            print("Warning g_error_handled during {} call. "
```

Comment:
I think we should use `warning` module here. Not sure if raising an exception is better or not.